src/Core/Services/IUserService.cs

```diff
@@ -45,6 +45,7 @@ Task<IdentityResult> ChangeKdfAsync(User user, string masterPassword, string new
     Task<string> GenerateUserTokenAsync(User user, string tokenProvider, string purpose);
     Task<IdentityResult> DeleteAsync(User user);
     Task<IdentityResult> DeleteAsync(User user, string token);
+    Task<IEnumerable<(Guid UserId, string? ErrorMessage)>> DeleteManyAsync(IEnumerable<User> users);
```

Comment:
You need to enable nullable reference types

```suggestion
#nullable enable
        Task<IEnumerable<(Guid UserId, string? ErrorMessage)>> DeleteManyAsync(IEnumerable<User> users);
#nullable disable
```